weareinreach/inreach-catalog
# update verification badge tooltip to include a link to the vetting process

## Description
update verification badge tooltip to include a link to the vetting process

## Asana ticket:
https://app.asana.com/0/1132189118126148/1202469970516633

## PR Checklist




## How to Test
1. hoover over the verification checkmark icon
2. the text should include a 'click here' link
3. clicking the link should open a new browser tab for https://inreach.org/vetting-process/
